src/Estimators/EstimatorManagerInput.h

```diff
@@ -39,15 +39,16 @@ class EstimatorManagerInputTests;
 class SpinDensityInput;
 class MomentumDistributionInput;
 class OneBodyDensityMatricesInput;
-using EstimatorInput  = std::variant<MomentumDistributionInput, SpinDensityInput, OneBodyDensityMatricesInput>;
+using EstimatorInput =
+    std::variant<std::monostate, MomentumDistributionInput, SpinDensityInput, OneBodyDensityMatricesInput>;
```

Comment:
Any specific reason to use monostate? Do we need to allow an input object state not associated with any estimator?

Reply:
If you don't want the leading variant element to be trivial constructable but the variant must be then you use monostate.

Reply:
I see. It seems adding a protection.